packages/typography/src/styled/StyledCode.ts

```diff
@@ -29,14 +52,15 @@ const colorStyles = (props: IStyledCodeProps & ThemeProps<DefaultTheme>) => {
 };
 
 interface IStyledCodeProps extends Omit<IStyledFontProps, 'size'> {
-  hue?: string;
+  hue?: ICodeProps['hue'];
```

Comment:
`hue?: string;` was causing confusion. So I refined the type to emphasize that `Code`, `StyledCode`'s only consumer, strictly supports `grey`, `red`, `green`, `yellow`, and `undefined`. 

https://github.com/zendeskgarden/react-components/blob/327ea52612f08bf0861d23853827cc2d8a590a7d/packages/typography/src/types/index.ts#L10

Passing an unsupported hue would trigger a TS error or an invalid `PropType` error at runtime.